api/v2/Routes/isadmin/index.js

```diff
@@ -0,0 +1,21 @@
+import { Router } from 'express';
+import 'express-async-errors';
+import { isAdmin } from '../../../lib/userlib.mjs';
+import { getEmailFromAuth } from '../../../lib/googleAuthHelper.mjs';
+
+const router = Router({ mergeParams: true });
+
+// Responds with whether or not the current user is an admin
+router.get('/', async (req, res) => {
+    try {
+        const authEmail = await getEmailFromAuth(req.headers['authorization']);
```

Comment:
This can throw an error if the user doesn't pass in an authorization header on this request.  Also, the header value for Authorization is capitalized which generally matters with headers, so make sure it's capitalized

Reply:
> Also, the header value for Authorization is capitalized which generally matters with headers, so make sure it's capitalized
Are you sure about this? I tested the API with "Authorization" instead and isAdmin returns false.
![image](https://github.com/Connor-Bernard/gradeView/assets/48500458/936096de-ddaa-445b-b85a-fe122340eb86)
![image](https://github.com/Connor-Bernard/gradeView/assets/48500458/30fd0fb7-5e69-47f2-bea5-79eebef6cf7b)

Reply:
This returns True if I change it back to "authorization"

Reply:
The standard header is capitalized.  Can you locate where that authorization in lower case is being thrown from.  It needs to be fixed, but maybe we do this in a separate PR

Reply:
Let's do this in a separate PR